.github/workflows/backport-pull-request.yml

```
@@ -0,0 +1,34 @@
+# This workflow is triggered when a pull request is merged and the label 'release' is present.
+# It opens a pull request to backport the changes from main to develop.
+name: Create backport pull request
+
+on:
+  pull_request:
+    types:
+      - closed
```

I think this will ensure we only run this workflow when it's necessary
```suggestion
  pull_request:
    branches:
      - main
    types:
      - closed
```

---

Thanks, suggestion committed :)